homeassistant/components/notify/apns.py

```diff
@@ -12,17 +12,27 @@
 from homeassistant.config import load_yaml_config_file
 from homeassistant.components.notify import (
     ATTR_TARGET, ATTR_DATA, BaseNotificationService)
+from homeassistant.const import CONF_NAME, CONF_PLATFORM
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers import template as template_helper
 
 DOMAIN = "apns"
```

Comment:
This was used to register a service under "apns". See https://home-assistant.io/components/notify.apns/. It doesn't seem to be normal procedure. In the docs it says that it will register two services but I only see one registration in the platform code. Not really sure how this is supposed to work. I'll have to read some more.